src/pages/Auth/Signup.jsx

```diff
@@ -29,6 +30,15 @@ const Signup = () => {
                 email:userObj.email,
                 password:userObj.password,
             });
+            if(response.status===201){
+                localStorage.setItem("user",JSON.stringify(response.data.foundUser));
+                localStorage.setItem("eToken",response.data.encodedToken);
+                seteToken(()=>response.data.encodedToken)
+                navigate("/")
+            }
+            else{
+                navigate("/signup");
+            }
             console.log(response);
```

Comment:
If console.log is not required here remove it

Reply:
okeiy will remove it,while refactor.
thanks for review!